data/saas/config/hubspot_config.yml

```diff
@@ -10,13 +10,12 @@ saas_config:
 
   client_config:
```

Comment:
In order to make the `client_config` consistent with the notation we use for `SaaSRequest`, I updated `client_config` to also use value placeholders (`<domain>`,`<api_key>`). The placeholder values for authentication can only come from `connector_params` so there is no need to define that here.